packages/nextjs/components/scaffold-eth/Input/InputBase.tsx

```diff
@@ -50,7 +50,7 @@ export const InputBase = <T extends { toString: () => string } | undefined = str
     <div className={`flex border-2 border-base-300 bg-base-200 rounded-full text-accent ${modifier}`}>
       {prefix}
       <input
-        className="input input-ghost focus-within:border-transparent focus:outline-none focus:bg-transparent focus:text-gray-400 h-[2.2rem] min-h-[2.2rem] px-4 border w-full font-medium placeholder:text-accent/50 text-gray-400"
+        className="input input-ghost focus-within:border-transparent focus:outline-none focus:bg-transparent h-[2.2rem] min-h-[2.2rem] px-4 border w-full font-medium placeholder:text-accent/80 text-base-content/90 focus:text-base-content/90"
```

Comment:
Maybe make this all `/70` or `/60` lol `/90` still pops out too much

Reply:
Changed all to `/70`, when less than 70 it becomes difficult to read the text, at least for me

<img width="619" alt="Screenshot 2024-11-21 at 15 59 10" src="https://github.com/user-attachments/assets/98c66f37-d2c3-497b-9527-966cd1ad8400">

<img width="642" alt="Screenshot 2024-11-21 at 15 59 29" src="https://github.com/user-attachments/assets/3c06adba-4d6f-410e-adea-b5e53fa5d551">

Reply:
Yeah for me 70 is ok, looking at pre-PR styles they could look like a disabled field.